nh_eobs/models/nh_clinical_wardboard.py

```diff
@@ -21,6 +21,10 @@ def _get_transfer_in_datetime(self):
             ward_moves = self.move_ids.filtered(
                 lambda move: move.location_id.usage == 'ward'
             )
+            # In theory there should never be a wardboard with no move IDs but
+            # in reality this situation occurs due to unrealistic demo data.
+            if not ward_moves:
+                return
```

Comment:
Could we raise (if we haven't done it already) some tech debt to look at the demo data and make it more representative, as to avoid having to add code to the application to deal with scenarios that should not happen? Then we can delete this if it's not needed anymore.

Reply:
Yeah sure, created [EOBS-2599](https://github.com/NeovaHealth/openeobs/pull/186/files).